tests/product/base_product_case.py

```diff
@@ -155,11 +155,11 @@ def build_installer_in_docker(self):
             shutil.copytree(
```

Comment:
trying to comment on line 137... anyway, can you add the uuid to this container too?